include/kernels/staggered_quark_smearing.cuh

```diff
@@ -29,9 +33,12 @@ namespace quda
 
     using real = typename mapper<Float>::type;
 
-    F out;           /** output vector field */
-    const F in;      /** input vector field */
-    const F in_pack; /** input vector field used in packing to be able to independently resetGhost */
+    static constexpr unsigned int max_n_src = MAX_MULTI_RHS;
+    const int_fastdiv n_src;
```

Comment:
This doesn't seem to be set/used anywhere

Reply:
[9a98546](https://github.com/lattice/quda/pull/1465/commits/9a98546a30b343a906a101d89215ab3c8322775c)